### PR TITLE
docs: Add documentation for the @poap-xyz/frames package

### PR DESCRIPTION
**Pull request by reobin**
## Description

Add docs pages for `@poap-xyz/frames`.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update (if none of the other choices apply)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/poap-xyz/poap.js/blob/main/documentation/CONTRIBUTING.md) document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Review by jm42 — APPROVED** [review]
The preview links doesn't show the preview with this documentation or is broken or something is missing to be shown.

**Review by rlajous — CHANGES_REQUESTED** [review]
We need to add the configuration
https://nextra.site/docs/docs-theme/page-configuration

**Comment by reobin** [comment]
@rlajous Added the package, and it's better now.

File: docs/pages/packages/_meta.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "poaps": "Poaps",
```
Comment (jm42) [inline docs/pages/packages/_meta.json:2]: POAP in style:

```suggestion
  "poaps": "POAPs",
```

**Review by rlajous — APPROVED** [review]

**Pull request merged**